### PR TITLE
publish release and develop changes. Updates logback configuration

### PR DESCRIPTION
**Pull request by Lesrac**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -184,3 +185,30 @@ project.afterEvaluate {
         }
     }
 }
+
+tasks.register("publishVersion") {
```
Comment (rsteppac) [inline build.gradle.kts:189]: What is the task actually doing? Could it be that the task logic is missing?

File: src/main/resources/logback-spring.xml
Comment (rsteppac) [inline src/main/resources/logback-spring.xml:1]: What is the rational behind logging JSON encoded logs to stdout and pipe separated logs to a file? Would it not be better to use the same human friendly pipe separated format for both appenders and rather document how to use a custom logback.xml with the client with an example of the JSON encoder?

**Review by rsteppac — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (Lesrac) [reply build.gradle.kts:189]: It triggers the MavenPublication "bootJava" (see usage in workflows/publish.yml). But yeah it's not really obvious that the "dependsOn" trigger that one

File: src/main/resources/logback-spring.xml
Comment (Lesrac) [reply src/main/resources/logback-spring.xml:1]: I assume that the stdout is used to be processed by another service (like Azure Application Insights or Splunk Universal Forwarder) and therefore the JSON encoding would be the preferred way. I wouldn't know another reason outside of development that would prefer human friendly readable stdout logs

**Pull request merged**